open-company/open-company-web
# Slack selector component

card: https://trello.com/c/vSNRM6c1

Test:
- run `lein test-ui-components!`
- open a report page
- check at the bottom the rendered selector
- try to select a different value
- try the search field
